build.yaml

```diff
@@ -1584,6 +1584,17 @@ steps:
            --verbose \
            site/www/* \
            /io/www
+
+     # Grab the latest release of the docs
+     LATEST_02_SHA=$(gsutil cat gs://hail-common/builds/0.2/latest-hash/cloudtools-5-spark-2.4.0.txt 2>/dev/null)
+     gsutil cat gs://hail-common/builds/0.2/docs/hail-0.2-docs-$LATEST_02_SHA.tar.gz |
+         tar zxvf - -C /io/www --strip-components=1
```

Comment:
Should this be /docs/0.2 like 0.1 below?

Reply:
addressed above